test/integration/src/test/java/io/pravega/test/integration/ControllerStreamMetadataTest.java

```diff
@@ -189,4 +193,17 @@ public void streamManagerImpltest() {
         // Delete twice
         assertFalse(streamManager.deleteScope(SCOPE));
     }
+
+    @Test
+    public void testListCompletedTxns() {
+        // Create test scope. This operation should succeed.
+        assertTrue(controller.createScope(SCOPE).join());
+
+        assertTrue(controller.createStream(SCOPE, STREAM, streamConfiguration).join());
+
+        TxnSegments txnSegments = controller.createTransaction(Stream.of(SCOPE, STREAM), 15000L).join();
+
+        List<TransactionInfo> listUUID = controller.listCompletedTransactions(Stream.of(SCOPE, STREAM)).join();
+        assertEquals(0, listUUID.size());
```

Comment:
This is fine, but why don't you commit some transactions and test that you get the right IDs for the completed transactions? Also, can you add in this test what happens if you have more than 200 transactions completed? Is the limit respected?

Reply:
Updated integration test and also added API call in system test.

Reply:
Thanks for adding new tests. It may be good to also add  a test where there are > 100 completed txns and only the last 100 are returned. This test case is missed.

Reply:
Updated integration test as well as Junit test case to verify no of records returned by API. Now this API can return maximum maximum 500 records.